backend/src/BuiltinExecution/Libs/Json.fs

```diff
@@ -84,7 +84,7 @@ let rec serialize
       else if System.Double.IsPositiveInfinity f then
         w.WriteStringValue "Infinity"
       else
-        let result = sprintf "%.12g" f
+        let result = sprintf "%.16g" f
```

Comment:
Changing the float formatting precision to 16 makes it consistent with other languages. Is that sufficient, or do we want to move away from sprintf entirely?